src/users/users.service.spec.ts

```diff
@@ -321,15 +321,6 @@ describe('UsersService', () => {
         expect(ids).toContain(userA.id);
         expect(ids).not.toContain(userB.id);
       });
```

Comment:
Minor feature loss that would make the code much less clean, open to suggestions.